wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -24,16 +26,40 @@ namespace :wagn do
 
     # Note Cardio based, but some use cases need this in app dir.  Solution?
     ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
-     
+
     puts "creating"
     Rake::Task['db:create'].invoke
 
     puts "loading schema"
     Rake::Task['db:schema:load'].invoke
-    
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "clear and load fixtures with existing tables"
+  task :create do
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
+    Rake::Task['wagn:clear'].invoke
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "empty the card tables"
+  task :clear => :environment do
+    conn = ActiveRecord::Base.connection
+
+    puts "delete all data in bootstrap tables"
+    WAGN_BOOTSTRAP_TABLES.each do |table|
+      conn.delete "delete from #{table}"
+    end
+  end
+
+  desc "Load bootstrap data into database"
+  task :seed do
```

Comment:
from the command line `wagn seed` calls `rake wagn:create`.  This is confusing!  

Since `wagn seed` is the most public part of the api (see https://github.com/wagn/wagn/blob/master/README.rdoc#3-create--seed-database), perhaps we could make it so that "seed" means "create and pre-populate".

In other words, `wagn seed` would call `rake wagn:seed` which would call the :create and :prepopulate tasks.